test/vtep/vxlan-responder.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
```

Comment:
I think we've generally defined this step like this in the repository:

```suggestion
#!/usr/bin/env python3
```

Reply:
done